src/db/tables/subscribers.ts

```diff
@@ -429,6 +414,85 @@ async function getSubscribersWaitingForMonthlyEmail(
 }
 /* c8 ignore stop */
 
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function getFreeSubscribersWaitingForMonthlyEmail(): Promise<
+  SubscriberRow[]
+> {
+  const flag = await getFeatureFlagData("MonthlyReportFreeUser");
+  const accountCutOffDate = parseIso8601Datetime(
+    process.env.MONTHLY_ACTIVITY_FREE_EMAIL_ACCOUNT_CUTOFF_DATE ??
+      "2022-01-01T00:00:00.000Z",
+  );
+
+  if (!flag?.is_enabled) {
+    return [];
+  }
+
+  let query = knex("subscribers")
+    .select<SubscriberRow[]>("subscribers.*")
```

Comment:
One thing I should point out here: I did not add a LIMIT clause. The reason for this is that we also need to select people who had en-US as their sign up language, but we need to parse the accept-language field to do so. So this'll fetch a lot of records, and then only afterwards filter them down to a small batch. (I should probably add this as a comment, but I'm on my phone now.)

Reply:
Yes please add this in a comment. Let me know if you have problems or need help optimizing this, it might help to look at the SQL generated here. Or we could just use the GCP SQL debugging tools to inspect once it lands on stage (we can also see the actual generated SQL there too).